### DISCUSSION TITLE
Control which Auxvariable to be saved in exodus file

### DISCUSSION THREAD

**Discussion by Traiwit** [idaholab/moose#28286] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hi all,

So, in general case all Auxvariables will be saved into .e file during outputting step.

I just wondering is there a way to control which Auxvariable to be saved in exodus file ?

for example, I want some of the Auxvariables to be saved every 10 timestep, but the rest to be saved at every timestep? is it possible to do this?

or just choose which Auxvariables to be saved?

Thank you guys!
Traiwit

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can't choose the frequency for each auxvariable currently in a single exodus file

You can choose which variables using hide/show in the Outputs block
And you can choose the output frequency using the interval parameters. 

You could have two exodus output for the two different output frequency groups of variables

Guillaume

**Reply by Traiwit** [reply]
Hi @GiudGiud,

thank you! I will try to work around this

a quick question: for hide/show, if I only input some variable under hide, will it still show the rest of the auxvariable OR if I choose to do `hide`, I will also need to mention all the `show`?

I will get back to you RE: interval parameters soon.

Traiwit

**Reply by GiudGiud** [reply]
If you use hide you don't need to use show. And vice versa
